.github/workflows/test-os.yml

```diff
@@ -165,6 +171,7 @@ jobs:
           name: test-reports-${{ env.TEST_REPORT_NAME }}
           path: ./bin/testreports
           retention-days: 1
+          overwrite: true
```

Comment:
Instead of overwrite you could add OS to the report name and separate server 2022 and server 2025 artifacts